Graphics/Blank.hs

```diff
@@ -99,8 +99,11 @@ module Graphics.Blank
          -- ** 'CanvasContext', and off-screen Canvas.
         , CanvasContext
         , newCanvas
-        , newAudio --NickS addition
-        , AudioInfo -- NickS addition
+          -- ** Audio functionality
+        , newAudio 
+        , AudioInfo
+        , play
+        , pauseAud
```

Comment:
There some inconsistency in the naming conventions of audio-related functions right now. For consistency with the image-related functions, let's use the word `Audio`, fully spelled out, with these names (i.e., name them `playAudio` and `pauseAudio`).